events/babylon_infamy_rebalance_events.txt

```diff
@@ -915,3 +877,95 @@ infamy_rebalance.15 = {
 		}
 	}
 }
+
+infamy_rebalance.16 = {
+	# Double down possibility
+	type = country_event
+	event_image = {
+		video = "gfx/event_pictures/unspecific_signed_contract.bk2"
+	}
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/unspecific/signed_contract"
+	icon = "gfx/interface/icons/event_icons/event_newspaper.dds"
+	title = infamy_rebalance.16.t
+	desc = infamy_rebalance.16.d
+	flavor = infamy_rebalance.16.f
+	duration = 1
+	immediate = {
+		# save pariah scope
+		random_country = {
+			limit = {
+				has_modifier = modifier_pariah_international_isolation
+				NOT = {
+					root = {
+						has_diplomatic_pact = {
+							who = prev
+							type = embargo
+						}
+					}
+				}
+			}
+			save_scope_as = pariah_scope
+		}
+		set_variable = {
+			value = scope:pariah_scope.infamy
```

Comment:
Ehm.. you get more infamy than the pariah? Am I reading this correctly?

Reply:
No, you get a third of what they have. Could do a divide = 3, but I figured doing it percentage based would be easier to maintain

Reply:
aah whoops somehow I thought it was an add later...